### PR TITLE
Insert consistency command

### PR DESCRIPTION
**Pull request by willronchetti**
- Add a simple command `check-insert-consistency` that will catch and report certain types of inconsistencies with ``master-inserts`` and data in live environments

### CODE REVIEW

File: src/encoded/commands/check_insert_consistency.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+def main():
+    parser = argparse.ArgumentParser(description="Checks consistency of inserts with a live env")
+    parser.add_argument("--env", help="Env to use (if not data)", default='data')
+    parser.add_argument("--item-type", help="Type to check (snake case)", default='all')
+    # TODO: Maybe do this? However, tricky because we're not always sure what
+    # the right fix is, and it could assume wrong. - Will 22 May 24
+    # parser.add_argument("--fix", action='store_true', default=False,
+    #                     help="Whether to fix the inserts")
+    args = parser.parse_args()
+    checker = InsertConsistencyChecker(args.env, args.item_type, args.fix)
```
Comment (dmichaels-harvard) [inline src/encoded/commands/check_insert_consistency.py:84]: Need to remove args.fix from arg list as it's commented out.

**Review by dmichaels-harvard — APPROVED** [review]
One minor fix.

**Comment by Bianca-Morris** [comment]
This is now working for me. Let me know if you want another look/if you add more features. Thanks again!

**Review by Bianca-Morris — APPROVED** [review]

**Comment by Bianca-Morris** [comment]
Actually, one additional note. I'm noticing that the new inserts on local are being added to the array where I'd expect to see only the inserts with active conflicts:

```
(smaht-portal) MBIPARLM1906951:smaht-portal bmorris$ poetry run check-insert-consistency --env smaht-wolf --item-type page
('Configuration file exists at /Users/bmorris/Library/Application Support/pypoetry, reusing this directory.\n\nConsider moving TOML configuration files to /Users/bmorris/Library/Preferences/pypoetry, as support for the legacy directory will be removed in an upcoming release.',)
SMaHTProject initialized with name 'smaht' and notable alias 'encoded'.
Found new uuid on local not present on https://wolf.smaht.org
Found new uuid on local not present on https://wolf.smaht.org
Found new uuid on local not present on https://wolf.smaht.org
Found new uuid on local not present on https://wolf.smaht.org
Found new uuid on local not present on https://wolf.smaht.org
Item Type: page
    The following uuids require further investigation:
        1. 508b5585-2329-4878-913b-4b70eeea6cb6
        2. 2aae05f6-595d-4b13-a1be-07294c5d48af
        3. 411bb4f5-d465-45c1-80b5-1e29e8190e5b
        4. fd724bbc-415c-46c6-b1b0-366da2dc990c
        5. 458bae26-17ba-4af9-adc5-9c7c4a13cbfd
```

I believe all of those listed are present on local, but not on wolf. On our end, it's pretty common to add a new insert, so it might make sense to have those listed under a different heading. Like "The following uuids are new on local branch:" and then the ones that are potential conflicts are listed as requiring further investigation.

Nothing major or blocking, just a UX note.

File: src/encoded/commands/load_data_from_local.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                            return result
+                        elif 'HTTPNotFound' in msg:
+                            #ff_utils.post_metadata(f'/{camelize(k)}', v)
+                            PRINT(f'Found existing item')
+                        else:
+                            PRINT(f'Other error encountered  - exiting: {msg}')
+                            return {}
+                    else:
+                        diff = DiffManager().diffs(current, v)
+                        import pdb; pdb.set_trace()
```
Comment (dmichaels-harvard) [inline src/encoded/commands/load_data_from_local.py:65]: Assuming presence of this pdb is remnant/oversight.

Comment (willronchetti) [reply src/encoded/commands/load_data_from_local.py:65]: Yeah just not complete yet

**Review by dmichaels-harvard — APPROVED** [review]

**Pull request merged**